kv/types.go

```diff
@@ -95,4 +95,7 @@ type Store interface {
 	// CheckAndSet stores the value for the given key if the current version
 	// matches the provided version
 	CheckAndSet(key string, version int, v proto.Message) (int, error)
+
+	// History returns the value for a key in version range [from, to)
+	History(key string, from, to int) ([]Value, error)
```

Comment:
GetBetween seems too vague, GetRange might sound like a function that lists all the keys in a range (which will be added soon)